db_helper.py

```diff
@@ -0,0 +1,21 @@
+import pyodbc
+
+
+class db_helper:
+
+    def __init__(self, csv_file_nm, sql_server_nm, db_nm, db_table_nm):
+        conn = self.connect_db(sql_server_nm, db_nm)
+        self.insert_data(conn, csv_file_nm, db_table_nm)
+        conn.close
+
+    def connect_db(self, sql_server_nm, db_nm):
+        conn_string = 'DRIVER={ODBC Driver 17 for SQL Server};SERVER=' + sql_server_nm + ';DATABASE=' + db_nm + ';Trusted_Connection=yes;'
+        conn = pyodbc.connect(conn_string)
+        return conn
+
+    def insert_data(self, conn, csv_file_nm, db_table_nm):
+        qry = "BULK INSERT " + db_table_nm + " FROM '" + csv_file_nm + "' WITH (DATAFILETYPE = 'char', FIRSTROW = 6, FIELDTERMINATOR = ',')"
```

Comment:
Where are you handling duplicates? 
Scenario : 
1. old data which has already inserted into sql server is now loaded in landing zone (csv file). Where are we handling this logic?